pdg/src/main.rs

```diff
@@ -361,7 +370,7 @@ mod tests {
     #[test]
     fn analysis_test_pdg_snapshot_debug() -> eyre::Result<()> {
         init();
-        let pdg = analysis_test_pdg_snapshot(Profile::Debug)?;
+        let pdg = analysis_test_pdg_snapshot(Profile::Debug, RuntimeKind::BackgroundThread)?;
```

Comment:
worth using `default()` instead?

Reply:
Yeah, good idea.

Reply:
Fixed in https://github.com/immunant/c2rust/pull/700/commits/7efe3f8cb2338a64712b66bded268c31de4397c2.